src/Sylius/Bundle/WebBundle/Resources/views/Backend/Inventory/macros.html.twig

```diff
@@ -72,7 +72,7 @@
                                     <td>
                                         <ul>
                                         {% for option in variant.options %}
-                                            <li><strong>{{ option.name }}</strong>: {{ option.value }}</li>
+                                            <li><strong>{{ option.presentation }}</strong>: {{ option.value }}</li>
```

Comment:
rename is simple. but the object level changes are more important. will update it later todat.